Notebooks/README.md

```diff
@@ -13,7 +13,13 @@ validation programs.
 
 | Notebook | Short description | Links | Owner  |
 |----------|-------------------|-------|--------|
-| DC2 Postage Stamps | Make some small cutout images and visualize them | [ipynb](Notebooks/DC2%20Postage%20Stamps.ipynb), [html](http://htmlpreview.github.io/?https://github.com/LSSTDESC/DC2_Repo/blob/html/Notebooks/DC2_Postage_Stamps.html) | [Michael Wood-Vasey](https://github.com/LSSTDESC/DC2_Repo/issues/new?body=@wmwv) |
-| LSST Stack matching code | Spatial matching of objects using the DM Stack | [ipynb](Notebooks/LSST_Stack_matching_code.ipynb), [html](http://htmlpreview.github.io/?https://github.com/LSSTDESC/DC2_Repo/blob/html/Notebooks/LSST_Stack_matching_code.html) | [Jim Chiang](https://github.com/LSSTDESC/DC2_Repo/issues/new?body=@jchiang87)   |
-| Plotting the Run 1.2 skymap | Visualize the tracts and patches on the sky | [ipynb](Notebooks/Plotting_the_Run1.1p_skymap.ipynb), [html](http://htmlpreview.github.io/?https://github.com/LSSTDESC/DC2_Repo/blob/html/Notebooks/Plotting_the_Run1.1p_skymap.html) | [Jim Chiang](https://github.com/LSSTDESC/DC2_Repo/issues/new?body=@jchiang87) |
-| DC2 Coadd Run1.1p Butler Access HSC selection | Use the Butler to apply the HSC object selection cuts | [ipynb](Notebooks/DC2_Coadd_Run1.1p_Butler_Access_HSC_selection.ipynb), [html](http://htmlpreview.github.io/?https://github.com/LSSTDESC/DC2_Repo/blob/html/Notebooks/DC2_Coadd_Run1.1p_Butler_Access_HSC_selection.html) | [Jim Chiang](https://github.com/LSSTDESC/DC2_Repo/issues/new?body=@jchiang87) |
+| DC2 Reference Catalog Reader | Simple example of reading the reference catalog | [ipynb](DC2%20Reference%20Catalog%20Reader.ipynb), [rendered](https://nbviewer.jupyter.org/github/LSSTDESC/DC2_Repo/blob/rendered/Notebooks/DC2_Reference_Catalog_Reader.nbconvert.ipynb), [log](https://github.com/LSSTDESC/DC2_Repo/blob/rendered/Notebooks/log/DC2_Reference_Catalog_Reader.log)  | [Anze Slosar](https://github.com/LSSTDESC/DC2_Repo/issues/new?body=@slosar) |
```

Comment:
Recently I was working on a more complicated table than this for the imSim wiki in markdown.  Since every row of the table has to be a single line just connected with "|" it is really hard to edit parse when you are adding long text.

What I found worked better in this case was to write just this page in reStructured text instead.  GH parses those files too.

Here is the page for reference.  
https://github.com/LSSTDESC/imSim/wiki/Feature-Matrix

A table entry looks like this:

```
.. list-table::
   :widths: 10 10 10 15 15
   :header-rows: 1

   * - Effect
     - Implementation
     - Data / Model Source
     - Short description
     - Validation Page and Notebooks

   * - Brighter Fatter
     - GalSim Feature (link)
     - BF slopes measured on TS-X See LCA-XXX
     - GalSim reads in output from full electrostatic solver and does linear interpolation,  See page link
     - `Page link <https://github.com/LSSTDESC/imSim/wiki/brighter_fatter_validation>`_
       The solution to a full electrostatic solver interpolates distorted pixel boundaries as
       function of pixel level. These boundaries are used as the sensor is being filled with electrons.
```

Which is a lot easier for a human to edit.  You might want to consider doing this.  It would just take a few minutes to change I think.